src/main/java/Task.java

```diff
@@ -0,0 +1,44 @@
+public class Task {
+    protected String taskDescription;
+    protected boolean isDone;
+
+    private static int numberOfTasks = 0;
+
+    public Task(String taskDescription) {
+        this.taskDescription = taskDescription;
+        this.isDone = false;
+        numberOfTasks += 1;
+    }
+
+    public String getTaskDescription() {
+        return taskDescription;
+    }
+
+    public void setTaskDescription(String taskDescription) {
+        this.taskDescription = taskDescription;
+    }
+
+    public static int getNumberOfTasks()
+    {
+        return numberOfTasks;
+    }
+
+    public void markAsDone()
+    {
+        isDone = true;
+    }
+
+    public void markAsUndone()
+    {
+        isDone = false;
+    }
```

Comment:
might want to be consistent with whether you put the opening braces into its own line or at the end of the previous line.